addons/payment_custom/models/payment_provider.py

```diff
@@ -45,19 +45,24 @@ def _transfer_ensure_pending_msg_is_set(self):
             return
 
         account_payment = self.env['ir.module.module']._get('account_payment')
-        if account_payment.state != 'installed':
-            return
-
         for provider in transfer_providers_without_msg:
-            company_id = provider.company_id.id
-            # filter only bank accounts marked as visible
-            accounts = self.env['account.journal'].search([
-                ('type', '=', 'bank'), ('company_id', '=', company_id)
-            ]).bank_account_id
-            provider.pending_msg = f'<div>' \
-                f'<h3>{_("Please use the following transfer details")}</h3>' \
-                f'<h4>{_("Bank Account") if len(accounts) == 1 else _("Bank Accounts")}</h4>' \
-                f'<ul>{"".join(f"<li>{account.display_name}</li>" for account in accounts)}</ul>' \
-                f'<h4>{_("Communication")}</h4>' \
-                f'<p>{_("Please use the order name as communication reference.")}</p>' \
-                f'</div>'
+            if account_payment.state != 'installed':
+                provider.pending_msg = f'<div>' \
+                    f'<h3>{_("Please use the following transfer details")}</h3>' \
+                    f'<h4>{_("Bank Account")}</h4>' \
+                    f'<h4>{_("Communication")}</h4>' \
+                    f'<p>{_("Please use the order name as communication reference.")}</p>' \
+                    f'</div>'
```

Comment:
Could be assigned in batch to all the providers, since the values will be the same.

Reply:
thanks :)